greenery/parse.py

```diff
@@ -191,9 +191,9 @@ def match_charclass(string: str, i):
         pass
 
     # e.g. if seeing "\\t", return "\t"
-    for key in escapes.keys():
+    for ekey in escapes.keys():
```

Comment:
Is `key` a reserved name in Python these days?

Reply:
No, it was a local name bound to a different type in previous loop expressions in the same function.